src/gw_large_cell_gamma.F

```diff
@@ -1052,6 +1078,27 @@ SUBROUTINE compute_W_MIC(bs_env, qs_env, mat_chi_Gamma_tau, fm_W_MIC_time)
       CALL cp_cfm_release(cfm_V_sqrt_ikp)
       CALL dbcsr_deallocate_matrix_set(mat_chi_Gamma_tau)
 
+      ! Marek : Fourier transform W^MIC(itau) back to get it at a specific im.frequency point - iomega = 0
+      ! TODO : Should this be activeted here or somewhere else? Maybe directly in the main (gw) routine?
+      ! TODO : Add activation dependent on the rt_propagation
+      ! Create the matrix TODO: Add release statement in some well understood place - at destruction of bs_env?
+      IF (bs_env%rtp_method == rtp_method_bse) THEN
+         ! ! DEBUG : Print weights for smallest frequency point and for zero frequency
+         ! PRINT *, "Weights smallest", bs_env%weights_cos_t_to_w(1,:)
+         ! PRINT *, "Weights zero", bs_env%imag_time_weights_freq_zero(:)
```

Comment:
Wrap it or remove it.

Reply:
Same as above, I removed the commented lines